### DISCUSSION TITLE
How to scale a model

### DISCUSSION THREAD

**Discussion by cameron1024** [bevyengine/bevy#8438] [Q&A] [answered]
This seems like it should be really simple, but I'm having trouble finding any way of going about it.

I'm trying to replace my character (currently a cylinder) with a more sensible model, so I grabbed the fox example from the GLTF examples, but it's very small.

I was looking for a way to simply make it 100x bigger in all dimensions, but wasn't able to. I tried setting the `tranform.scale` field of `SceneBundle` but that didn't seem to have any effect.

I asked this question on Reddit, and was suggested to edit the file in blender, but this isn't suitable for my use case since I also would like to be able to dynamically change the size of the model based on game logic.

Is this possible?

**Comment by nicopap** [comment, marked as answer]
How are you changing the scale of the scene bundle? It should work as you describe it. If you set something like `transform: Transform::from_scale(Vec::splat(100.0))` it should work.

**Reply by cameron1024** [reply]
You're right, this was user error on my part.

I was setting the `scale` in an outer function, then passing the transform to another function which was overwriting the `scale` field, so my changes weren't actually changing anything.

Thanks for the reply 😊
